.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Contributing to `twilio-verify-passkeys-web`
+
+Thank you for your interest in contributing to `twilio-verify-passkeys-web`. We appreciate your help in making it better. Please take a moment to review these guidelines before getting started.
+
+- [Branches](#branches)
+- [Code of Conduct](#code-of-conduct)
+- [Question or Problem?](#question-or-problem)
+- [Issues and Bugs](#issues-and-bugs)
+- [Feature Requests](#feature-requests)
+- [Documentation Fixes](#documentation-fixes)
+- [Submission Guidelines](#submission-guidelines)
+- [Coding Rules](#coding-rules)
+- [License](#license)
+
+## Branches
+
+- **`main`**: The `main` branch is the stable branch containing production-ready code. It should always reflect the latest release.
+- **`dev`**: The `dev` branch is where all new features and changes should be directed. This branch might contain work-in-progress code.
+- Any other branch contains code with new features or improvements that are waiting to be merged into the `dev` branch.
+
+## Code of Conduct
+
+Help us keep `twilio-verify-passkeys-web` open and inclusive. Please be kind to and considerate of other developers, as we all have the same goal: make `twilio-verify-passkeys-web` as good as it can be.
+By interacting with the project in any shape or form you are agreeing to the project's [Code of Conduct](https://github.com/twilio/twilio-verify-passkeys-web?tab=coc-ov-file). If you feel like another individual has violated the code of conduct, please raise a complaint to [open-source@twilio.com](mailto:open-source@twilio.com).
+
+## Question or Problem?
+
+If you have questions about how to use `twilio-verify-passkeys`, please see our [docs](../README.md), and if you don't find the answer there, please contact [Twilio Support](https://www.twilio.com/help/contact) with any issues you have.
```

Comment:
Shouldn't it be `twilio-verify-passkeys-web`?

Reply:
Fixed